code/game/gamemodes/miniantags/guardian/types/lightning.dm

```diff
@@ -18,6 +18,17 @@
 	var/list/enemychains = list()
 	var/successfulshocks = 0
 
+/mob/living/simple_animal/hostile/guardian/beam/New(loc, mob/living/user, mob/living/user)
```

Comment:
:eyes: What I meant was that you needed to add a second `user` argument at this line further down in the file:
`var/mob/living/simple_animal/hostile/guardian/G = new pickedtype(user)` -> `new pickedtype(user, user)`